app/modules/booking/view_bookings/viewBookings.query.js

```diff
@@ -32,9 +31,22 @@ const viewBookingsQuery = async(searchKey, sortKey, sortOrder) => {
                     [Op.iLike]: `%${searchKey}%`
                 }
             }]
-        };
+        }
+
+    };
+    if (page, pagesize) {
+        const offset = page * pagesize
```

Comment:
If the page is 1 the offset should be 0, did you handle that case?
offset: req.query.page > 1 ? limit * (req.query.page - 1) : 0,